src/Sylius/Behat/Context/Api/Admin/ManagingCatalogPromotionsContext.php

```diff
@@ -230,6 +264,27 @@ public function thereShouldBeNewCatalogPromotionOnTheList(int $amount = 0): void
         Assert::count($this->responseChecker->getCollection($this->client->index()), $amount);
     }
 
+    /**
+     * @Then /^it should have "([^"]+)" discount$/
+     */
+    public function itShouldHaveDiscount(string $amount): void
+    {
+        $catalogPromotion = $this->responseChecker->getCollection($this->client->getLastResponse())[0];
+
+        Assert::same($catalogPromotion['actions'][0]['configuration']['amount'], $amount);
```

Comment:
Now the question is - should we use typical percentage notation (40% -> 0.4) or absolute values(40% -> 40)? For sure, we shouldn't use `40%` as a config value. This transformation should be done before assertion

Reply:
I would go with `0.4` as it's consistent with actions on cart promotions and is easier to be used for calculation